src/zoid/qr-code/component.jsx

```diff
@@ -208,7 +218,7 @@ export function QRCodeContainer({
                 position: absolute;
                 left: 8px;
                 content: ' ';
-                height: 16px;
+                height: 20px;
```

Comment:
Remove the close styling.

Reply:
Since we are moving the close button to the `prerenderTemplate`, maybe we will need to keep this style. I will need to work a bit on the code try check that. Is that ok?